src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -184,9 +184,6 @@ public sealed class SetEnvCommandExtension : RunnerService, IActionCommandExtens
 
         public void ProcessCommand(IExecutionContext context, string line, ActionCommand command, ContainerInfo container)
         {
```

Comment:
should this method only has one line?
```
throw new Exception(String.Format(Constants.Runner.UnsupportedCommandMessageDisabled, this.Command));
```

Reply:
No because we want users to be able to set `ACTIONS_ALLOW_UNSECURE_COMMANDS` to use the old commands if needed, some actions may not have updated already and giving users a workaround is helpful until they do get updated!

Reply:
confirmed with @chrispat 
APPROVED.